operational_analysis/methods/turbine_long_term_gross_energy.py

```diff
@@ -121,13 +121,15 @@ def run(
          reanal_subset(:obj:`list`): Which reanalysis products to use for long-term correction
          uncertainty_scada(:obj:`float`): uncertainty imposed to scada data (used in UQ = True case only)
          max_power_filter(:obj:`tuple`): Maximum power threshold (fraction) to which the bin filter
-                                         should be applied (default 0.85). This should be a tuple in the UQ = True case,
+                                         should be applied (default is the interval between 0.8 and 0.9). 
```

Comment:
The docstring indents just need to be two spaces, so I would dedent that back to be in line with the "x" in "max". Same goes for the rest of the large indents in here.

Reply:
Thanks, I've fixed this!